### ISSUE TITLE
Reader: paid posts only feed

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#88456]
Paid posts feed should be available on both the app and the web.

Currently, in the app:

<img width="400" src="https://github.com/Automattic/wp-calypso/assets/87168/e2768686-f956-448a-8b19-2e68e58a3f7d">

In the web:

<img width="384" alt="image" src="https://github.com/Automattic/wp-calypso/assets/87168/657acfe3-5e14-4902-ac2f-930437a481eb">

**Comment by eoigal** [comment]
I might be missing something here but when I compare the mobile to web menu, it looks to me that the `Recent` option on web is called `Subscriptions` on mobile. This is something we can change on web to match mobile.

Is this issue suggesting we change that name or should we introduce a new stream that only streams paid-only content?

If its the latter, then I don't think it needs it's own stream, I think it should be a filter option under the 'Subscriptions' stream.

cc @simison

**Comment by simison** [comment]
> Is this issue suggesting we change that name or should we introduce a new stream that only streams paid-only content?

This.

I think a filter could work, too. I'm a bit describing solution but sentiment of getting quick access to content I'm paying is what matters most.

Filters for "image posts", "video posts", and "audio/podcast posts" would be helpful too.

**Comment by eoigal** [comment]
I had a look into how we can do this but unfortunately there isn't a simple solution (that I could find).

There is no way to differentiate feed types from the data structures used for the `userfeeds`, i.e. there isn't a way we can pass in some filter and adjust the query that retrieves the uses feeds to get a subset of feeds of a specific type, like paid only or video posts.

I suspect that if were to take on this project (and I can see the appeal) we would need to start by updating the `feedbag_item_info` table and adding a `feed_type` field. 

When we save a feed, we define the `feed_type` based on the post content.

We can then filter the Recent/Subscriptions feed stream based on the `feed_type`. 

We would need to back fill all the feeds currently saved which I assume will be a big project in of itself.

In short, I think this feature would improve the Reader UX but the underlying data structures don't immediately support it and it would take a non-trivial amount of work to get this data in place so that this feature can be supported.

**Comment by davemart-in** [comment]
Removing this from our current sprint as a result. Thanks for digging in @eoigal!
